home-data-migration/src/main/resources/db/changelog/0.0.1/Issue127.xml

```diff
@@ -13,6 +13,9 @@
             </column>
             <column name="COOPERATION_NAME" type="VARCHAR(70)">
             </column>
+            <column name="ENABLED" type="BOOLEAN">
```

Comment:
We cannot change existing migrations. We have to write new one with needed changes. Please read about how liquibase works.

Reply:
Okay, I got it, I will create a new one.